horovod/spark/common/params.py

```diff
@@ -23,6 +24,7 @@
 
 class EstimatorParams(Params):
     num_proc = Param(Params._dummy(), 'num_proc', 'number of processes')
+    data_module = Param(Params._dummy(), 'data_module', 'data module class to use when reading data')
```

Comment:
I suggest to move `data_module` filed under KerasEstimator since it is specific to KerasEstimator.

Refer the example of lightning estimator at https://github.com/horovod/horovod/blob/master/horovod/spark/lightning/estimator.py#L209